test/integration/singlecluster/scheduler/resourcetransformations/resource_transformations_test.go

```diff
@@ -0,0 +1,192 @@
+	ginkgo.BeforeAll(func() {
+		// Configure resource transformations for testing
+		transformations := []config.ResourceTransformation{
+			{
+				Input:    "nvidia.com/mig-1g.5gb",
+				Strategy: ptr.To(config.Replace),
+				Outputs: corev1.ResourceList{
+					"example.com/accelerator-memory": resource.MustParse("5Ki"),
+					"example.com/credits":            resource.MustParse("10"),
+				},
+			},
+			{
+				Input:      "nvidia.com/gpucores",
+				Strategy:   ptr.To(config.Replace),
+				MultiplyBy: "nvidia.com/gpu",
+				Outputs: corev1.ResourceList{
+					"nvidia.com/total-gpucores": resource.MustParse("1"),
+				},
+			},
+			{
+				Input:      "nvidia.com/gpumem",
+				Strategy:   ptr.To(config.Replace),
+				MultiplyBy: "nvidia.com/gpu",
+				Outputs: corev1.ResourceList{
+					"nvidia.com/total-gpumem": resource.MustParse("1"),
+				},
+			},
+		}
+		fwk.StartManager(ctx, cfg, managerAndSchedulerSetup(transformations))
+	})
+
+	ginkgo.BeforeEach(func() {
+		defaultFlavor = utiltestingapi.MakeResourceFlavor("default").Obj()
+		util.MustCreate(ctx, k8sClient, defaultFlavor)
+
+		ns = util.CreateNamespaceFromPrefixWithLog(ctx, k8sClient, "resource-transformations-")
+
+		clusterQueue = utiltestingapi.MakeClusterQueue("test-cq").
+			ResourceGroup(
+				*utiltestingapi.MakeFlavorQuotas("default").
+					Resource("example.com/accelerator-memory", "100Gi").
+					Resource("example.com/credits", "1000").
+					Resource("nvidia.com/gpu", "50").
+					Resource("nvidia.com/total-gpucores", "1000").
+					Resource("nvidia.com/total-gpumem", "102400").
+					Obj(),
+			).Obj()
+		util.MustCreate(ctx, k8sClient, clusterQueue)
+
+		localQueue = utiltestingapi.MakeLocalQueue("test-lq", ns.Name).ClusterQueue(clusterQueue.Name).Obj()
+		util.MustCreate(ctx, k8sClient, localQueue)
+	})
+
+	ginkgo.AfterEach(func() {
+		gomega.Expect(util.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
+		util.ExpectObjectToBeDeleted(ctx, k8sClient, clusterQueue, true)
+		util.ExpectObjectToBeDeleted(ctx, k8sClient, defaultFlavor, true)
+	})
+
+	ginkgo.AfterAll(func() {
+		fwk.StopManager(ctx)
+	})
+
+	ginkgo.It("should transform resources with MultiplyBy", func() {
+		ginkgo.By("Creating a workload with vgpu resources")
+		wl := utiltestingapi.MakeWorkload("multiply-wl", ns.Name).
+			Queue(kueue.LocalQueueName(localQueue.Name)).
+			Request("nvidia.com/gpu", "2").
+			Request("nvidia.com/gpucores", "20").
+			Request("nvidia.com/gpumem", "1024").
+			Obj()
+		util.MustCreate(ctx, k8sClient, wl)
+
+		ginkgo.By("Waiting for workload to be admitted")
+		util.ExpectWorkloadsToBeAdmitted(ctx, k8sClient, wl)
+
+		wlLookupKey := client.ObjectKeyFromObject(wl)
+		createdWorkload := &kueue.Workload{}
+
+		ginkgo.By("Verifying workload is created but not admitted yet", func() {
+			gomega.Eventually(func(g gomega.Gomega) {
+				g.Expect(k8sClient.Get(ctx, wlLookupKey, createdWorkload)).To(gomega.Succeed())
+			}, util.Timeout, util.Interval).Should(gomega.Succeed())
+		})
```

Comment:
I think this block is misleading, because above we already checked `ExpectWorkloadsToBeAdmitted`. Can we just remove it?